apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -350,6 +320,54 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'livechat/visitor.department.transfer',
```

Comment:
I'd also suggest using `livechat/visitor/department.transfer`

Following the current `resource/resource.action` pattern.